package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "bedrock-profile",
+  "version": "1.1.4-0",
+  "description": "Profiles for Bedrock Applications",
+  "main": "./lib",
+  "scripts": {},
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/bedrock-profile"
+  },
+  "keywords": [
+    "bedrock"
+  ],
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com/"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-profile/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-profile",
+  "dependencies": {
+    "assert-plus": "^1.0.0",
+    "axios": "^0.19.2",
+    "base64url-universal": "^1.0.0",
+    "bedrock-https-agent": "^1.0.1",
+    "bs58": "^4.0.1",
+    "cors": "^2.8.4",
+    "crypto-ld": "^3.5.2",
+    "did-method-key": "^0.4.0",
+    "edv-client": "^2.3.0",
+    "express-async-handler": "^1.1.4",
+    "jsonld-signatures": "^5.0.0",
+    "ocapld": "^1.6.1",
+    "webkms-client": "^2.0.1"
+  },
+  "peerDependencies": {
+    "bedrock": "1.12.1 - 3.x",
```

```suggestion
    "bedrock": "^3.0.0",
```
No reason to believe that this will work with anything other than 3.x